local-cli/runWpf/runWpf.js

```diff
@@ -1,45 +1,33 @@
 'use strict';
 
 const chalk = require('chalk');
-const build = require('./utils/build');
-const deploy = require('./utils/deploy');
+const automatedVS = require('../automated-vs/index');
+const WPFSolution = automatedVS.WPFSolution;
+const buildAndDeployUtils = require('../utils/buildAndDeployUtils');
+
+const minWinSDK = '8.1.0.0';
 
 function runWpf(config, args, options) {
-  // Fix up options
+
   options.root = options.root || process.cwd();
-  if (options.debug && options.release) {
-    console.log(chalk.red('Only one of "debug"/"release" options should be specified'));
-    return;
-  }
```

Comment:
there isn't a debug option, and things are configured for debugging by the absence of the release flag

Reply:
👍